packages/web-config-server/src/apiV1/RouteHandler.js

```diff
@@ -99,7 +99,7 @@ export class RouteHandler {
         const userPermissionGroups = await this.req.accessPolicy.getPermissionGroups(
           allCountryCodes,
         );
-        const userHasAccessToExcludedTypes = permissionGroups.every(permissionGroup =>
+        const userHasAccessToExcludedTypes = permissionGroups.some(permissionGroup =>
```

Comment:
Oh gosh, I forgot this functionality even existed. It's used by one project. I would be partial to us to scrapping it (not now though). I'll check with Georgia on the use case for it, and create a ticket to clean up if needed.